Core/src/org/sleuthkit/autopsy/datamodel/FileTypesByMimeType.java

```diff
@@ -439,10 +441,10 @@ private MediaSubTypeNodeChildren(String mimeType) {
         }
 
         @Override
-        protected boolean createKeys(List<Content> list) {
+        protected boolean createKeys(List<FileTypesKey> list) {
             try {
-                list.addAll(skCase.findAllFilesWhere(
-                        createBaseWhereExpr() + " AND mime_type = '" + mimeType + "'")); //NON-NLS
+                list.addAll(skCase.findAllFilesWhere(createBaseWhereExpr() + " AND mime_type = '" + mimeType + "'")
+                        .stream().map(f -> new FileTypesKey(f)).collect(Collectors.toList())); //NON-NLS
```

Comment:
to be even more concise you could use a method reference to the constructor:

...map(FileTypesKey::new).collect....